docs/gh_pages/sidebars.ts

```diff
@@ -66,6 +66,33 @@ const sidebars: SidebarsConfig = {
       id: "reports", // document ID
       label: "Reports", // sidebar label
     },
+    {
+        type: "category",
+        label: "Samples", // sidebar label
+        items: [
+            {
+              type: "category",
+              label: "Safe Loader Samples",
+              items: [
+                {
```

Comment:
Please see the screenshot for `link` type vs `doc` type

<img width="1021" alt="Screenshot 2024-07-16 at 11 15 18 PM" src="https://github.com/user-attachments/assets/493e75b8-3a46-489c-8779-59681f5289ed">